examples/requirements/src/language-server/requirements-and-tests-lang-module.ts

```diff
@@ -29,24 +33,26 @@ import { TestsLangModule, TestsLangServices } from './tests-lang-module';
  */
 export function createRequirementsAndTestsLangServices(context: DefaultSharedModuleContext): {
     shared: LangiumSharedServices,
-    RequirementsLang: RequirementsLangServices,
-    TestsLang: TestsLangServices
+    requirements: RequirementsLangServices,
+    tests: TestsLangServices
 } {
```

Comment:
Where do these changes come from? Was the former state out-dated? Did you re-generate the whole project with the yeoman generator?

Reply:
No, manual renaming for consistency with the other example languages.